cmd/ferretdb/main.go

```diff
@@ -360,42 +367,44 @@ func run() {
 
 	metricsRegisterer := setupMetrics(stateProvider)
 
-	startupFields := []zap.Field{
-		zap.String("version", info.Version),
-		zap.String("commit", info.Commit),
-		zap.String("branch", info.Branch),
-		zap.Bool("dirty", info.Dirty),
-		zap.String("package", info.Package),
-		zap.Bool("debugBuild", info.DebugBuild),
-		zap.Any("buildEnvironment", info.BuildEnvironment.Map()),
+	startupFields := []slog.Attr{
+		slog.String("version", info.Version),
+		slog.String("commit", info.Commit),
+		slog.String("branch", info.Branch),
+		slog.Bool("dirty", info.Dirty),
+		slog.String("package", info.Package),
+		slog.Bool("debugBuild", info.DebugBuild),
+		slog.Any("buildEnvironment", info.BuildEnvironment),
```

Comment:
How that line is formatted?

Reply:
It's `*types.Document`, so we would need `slog.LogValuer` implemented for this.

Reply:
Actually let's convert this to `*bson.Document`, that implements `slog.LogValuer` already https://github.com/FerretDB/FerretDB/blob/main/internal/bson/document.go#L220.

Reply:
Ignore my last comment, handling it in https://github.com/FerretDB/FerretDB/pull/4479